frontend/src/app/Settings/Users/ManageUsers.test.tsx

```diff
@@ -66,373 +140,312 @@ describe("ManageUsers", () => {
     );
     getUsers = jest.fn(() => Promise.resolve({ data: users }));
   });
-  it("displays the list of users and defaults to the first user", () => {
-    const { container } = render(
-      <MemoryRouter>
-        <ManageUsers
-          users={users}
-          loggedInUser={loggedInUser}
-          allFacilities={allFacilities}
-          updateUserPrivileges={updateUserPrivileges}
-          addUserToOrg={addUserToOrg}
-          deleteUser={deleteUser}
-          getUsers={getUsers}
-        />
-      </MemoryRouter>
-    );
-    expect(container).toMatchSnapshot();
-  });
-  it("disables logged-in user's settings", async () => {
-    const { container, findByText, getByText, getByLabelText } = render(
-      <MemoryRouter>
-        <ManageUsers
-          users={users}
-          loggedInUser={loggedInUser}
-          allFacilities={allFacilities}
-          updateUserPrivileges={updateUserPrivileges}
-          addUserToOrg={addUserToOrg}
-          deleteUser={deleteUser}
-          getUsers={getUsers}
-        />
-      </MemoryRouter>
-    );
 
-    fireEvent.click(getByText(displayFullName("Bob", "", "Bobberoo")));
-    await findByText("YOU");
-    expect(getByLabelText("admin", { exact: false })).toHaveAttribute(
-      "disabled"
-    );
-    expect(getByLabelText("user", { exact: false })).toHaveAttribute(
-      "disabled"
-    );
-    expect(getByLabelText("Entry only", { exact: false })).toHaveAttribute(
-      "disabled"
-    );
-    expect(container).toMatchSnapshot();
-  });
-  it("passes user details to the addUserToOrg function", async () => {
-    const store = mockStore({
-      organization: {
-        name: "Organization Name",
-      },
-      user: {
-        firstName: "Kim",
-        lastName: "Mendoza",
-      },
-      facilities: [
-        { id: "1", name: "Facility 1" },
-        { id: "2", name: "Facility 2" },
-      ],
+  describe("regular list of users", () => {
```

Comment:
The diff for this file is much more dramatic than what I actually did; the test assertions haven't changed. The actual changes to this file are as follows:
- added mocks and a `MockedProvider` to mock the `getUser` calls
- wrapped the render calls in `await waitFor()` (required because the component now makes a query directly, and without the `waitFor` React throws an error)
- extracted all the `render` calls to a top-level `describe`, since most of the tests have the same render, and added a `TestContainer` helper component to wrap the `MemoryRouter`, `Provider`, and `MockedProvider`